roles/default/kiali-deploy/templates/openshift/oauth.yaml

```diff
@@ -8,6 +8,5 @@ metadata:
     version: {{ kiali_vars.deployment.version_label }}
 redirectURIs:
   - {{ kiali_route_url }}
-  - localhost:3000
```

Comment:
This isn't part of the OpenID PR right? Are we sure this isn't needed and this won't break anything?

Reply:
It's not related to the OpenID work. This seems to be residuary code from previous dev-cycles. This seems to be part of a dev-setup and I think it shouldn't be here for better security.